Sources/Plasma/PubUtilLib/plNetGameLib/Intern.h

```diff
@@ -377,6 +377,26 @@ void NetTransCancelByConnId (unsigned connId, ENetError error);
 void NetTransCancelAll (ENetError error);
 void NetTransUpdate ();
 
+template<typename T, typename = void>
+struct HasTransId : std::false_type { };
+
+template<typename T>
+struct HasTransId<T, decltype((void)T::transId, void())> : std::true_type { };
+
+template<typename T>
+inline std::enable_if_t<HasTransId<T>::value, bool>
+NetTransRecvFromMsg(const uint8_t msg[], unsigned bytes, void*)
```

Comment:
Maybe this should be named something to indicates it's a generic catch-all?  Something like `NetTransRecvFromMsgGeneric`?